README.md

```diff
@@ -20,7 +20,7 @@ Deploy to vanilla k8s:
 ```
 $ git clone https://github.com/openshift-psap/special-resource-operator
 $ cd special-resource-operator
-$ make deploy TAG=master PLATFORM=k8s
```

Comment:
I think that if you are simplifying it, we should deploy `master` tag by default.

It is not a good UX to have a `make deploy` that doesn't work out of the box (cause the custom images don't exist)

Reply:
`TAG` is the output of `git branch --show-current` so it should work just out of the box (after clone).